db/0.schema.sql

```diff
@@ -14,9 +14,9 @@ CREATE TYPE hole AS ENUM (
 );
 
 CREATE TYPE lang AS ENUM (
-    'bash', 'brainfuck', 'c', 'fortran', 'go', 'haskell', 'j', 'java',
-    'javascript', 'julia', 'lisp', 'lua', 'nim', 'perl', 'php', 'python',
-    'raku', 'ruby', 'rust', 'swift'
+    'bash', 'brainfuck', 'c', 'csharp', 'fortran', 'fsharp', 'go', 'haskell',
```

Comment:
Any reason we couldn't use a literal sharp in the IDs/names? Does it look weird in the URL? I'm not wild about `csharp`

Reply:
I started with `c#` and I can't remember exactly what went wrong but there was something that happened.

Also, I'm not sure that would work for URLs like `https://code-golf.io/scores/all-holes/c#`, because it might be confused with the hash. If you don't think this would be a problem, I'll try it again.

We could try `c♯`, `f♯` but that looks weird to me because nobody uses it.

Reply:
I think this was the problem:
```
$ docker build -t codegolf/langs-c# .
invalid argument "codegolf/langs-c#" for "-t, --tag" flag: invalid reference format
See 'docker build --help'.
$ docker build -t codegolf/langs-c♯ .
invalid argument "codegolf/langs-c♯" for "-t, --tag" flag: invalid reference format
See 'docker build --help'.
```

Reply:
If you want we can try `c#` everywhere but the langs folder. But would we be able to get `https://code-golf.io/scores/all-holes/c#` to work?

Reply:
Oh I definitely meant a real sharp, so yeah `♯`, but failing that maybe `c-sharp` would read nicer?

Reply:
I'm not really sure if any of this matters, I'm just being picky and I'm not sure which I prefer, feel free to change nothing, either way I'll merge something tomorrow.

Reply:
I will contemplate it. If it makes C# and F# appear sequentially in the language ordering, I will probably like it.

Reply:
Looks like hyphen would keep them both ahead of fortran, sharp wouldn't
```
$ perl -E 'say for sort <{c,f}{{,-}sharp,♯} fortran>'
c-sharp
csharp
c♯
f-sharp
fortran
fsharp
f♯

Reply:
I know it was inspired by the musical sharp, but according to the language specification:

The name C# is pronounced “CSharp”. The name C# is written as the LATIN CAPITAL LETTER C (U+0043) followed by the NUMBER SIGN # (U+0023).

I'll try c-sharp and see how I like it.

Reply:
I'll try c-sharp then.

Reply:
Yeah on balance the musical sharp is a bit weird, and no one spells the language that way, I like the hyphen.

Reply:
I like it too.